netaddr.go

```diff
@@ -86,6 +86,37 @@ func ParseIP(s string) (IP, error) {
 	return IP{v6}, nil
 }
 
+// IPAddr returns the net.IPAddr representation of an IP. The returned value is
+// always non-nil, but the IPAddr.IP will be nil if ip is the zero value.
+// If ip contains a zone identifier, IPAddr.Zone is populated.
+func (ip IP) IPAddr() *net.IPAddr {
+	switch ip := ip.ipImpl.(type) {
+	case nil:
+		// Nothing to do.
+		return &net.IPAddr{}
+	case v4Addr:
+		return &net.IPAddr{
+			// Assume the caller wants the 4-byte representation.
+			IP: net.IPv4(ip[0], ip[1], ip[2], ip[3]).To4(),
```

Comment:
That's a little indirect, but I guess.

We could also simplify it in a follow-up change to just return the 4 bytes.